extensionsGallery.json

```diff
@@ -3419,6 +3419,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "73",
+					"extensionName": "sql-server-diagnostic-book",
+					"displayName": "SQL Server Diagnostic Book",
+					"shortDescription": "A collection of notebooks to troubleshoot and diagnose any version of SQL Server",
+					"publisher": {
+						"displayName":"Emanuele Meazzo",
+						"publisherId": "emanuelemeazzo",
+						"publisherName":"emanuelemeazzo"
+					},
+					"versions": [
+				  		{
+							"version": "0.0.1",
+							"lastUpdated": "3/9/2020",
```

Comment:
Flip the month and day here to keep the format the same as the rest of the dates in this file

Reply:
Switched to m/d/y